### PR TITLE
feat(login): removing session mangement

### PR DESCRIPTION
**Pull request by Rashad-j**

### CODE REVIEW

File: hexonet/ispapicli/modules/core.py [hunk 1]
```diff
@@ -26,6 +26,11 @@ def __init__(self):
         self.cl.setUserAgent("ISPAPICLI", __version__)
         # init db
         self.dbObj = DB()
```
Comment (papakai) [inline hexonet/ispapicli/modules/core.py:28]: If I remember correctly, the DB Object/Instance was just used as a way of Session Handling over DB. If so, wouldn't it then make sense to cleanup self.dbObj and the dependency DB? Please check this.

**Review by papakai — CHANGES_REQUESTED** [review]

File: hexonet/ispapicli/modules/core.py (hunk 1 above)
Comment (Rashad-j) [reply hexonet/ispapicli/modules/core.py:28]: The DB used in a broader sense. For commands as well as subusers autocomplete.

**Review by papakai — CHANGES_REQUESTED** [review]
Resolve conflicts please (rebase to master).

**Comment by Rashad-j** [comment]
please merge so I can resolve the github actions issue - the md linter issue

**Review by papakai — APPROVED** [review]

**Pull request merged**